front/src/components/todo/todo-list.vue

```diff
@@ -1,30 +1,48 @@
 <template>
   <div class="row">
-    <div class="flex column items-end">
+    <div class="flex column items-end full-width">
       <div class="flex full-width q-pt-sm items-center justify-between">
         <search-page></search-page>
-        <todo-create-button :createTodo='createTodo'></todo-create-button>
+        <todo-create-button :createTodo="createTodo"></todo-create-button>
+      </div>
+      <div v-if="!todos" class="full-width">
+        <NothingHere :text="text" :image="imgUrl" />
```

Comment:
Can you please make the component name in this-format instead of ThisFormat? For consistency with other components.